generator/all_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main_test
+
+import (
+	"errors"
+	"os/exec"
+	"testing"
+)
+
+func TestStaticCheck(t *testing.T) {
+	rungo(t, "run", "honnef.co/go/tools/cmd/staticcheck@v0.5.1", "./...")
+}
+
+func TestUnparam(t *testing.T) {
+	rungo(t, "run", "mvdan.cc/unparam@v0.0.0-20240917084806-57a3b4290ba3", "./...")
+}
+
+func TestVet(t *testing.T) {
+	rungo(t, "vet", "-all", "./...")
+}
+
+func TestGoModTidy(t *testing.T) {
+	rungo(t, "mod", "tidy")
```

Comment:
Fixed with `go mod tidy -diff`. I removed `go fmt` and left it in GitHub action because it always writes. We can update to use `gofmt` instead in the future if it feels worthwhile to move.